operator/config/samples/core_v1alpha1_transactioninvoke_zeto_register_anon.yaml

```diff
@@ -23,12 +23,12 @@ spec:
     {
       "name": "Zeto_Anon",
       "implementation": {
-        "implementation":        "{{ index .status.resolvedContractAddresses "zeto-impl-anon" }}",
-        "depositVerifier":       "{{ index .status.resolvedContractAddresses "zeto-g16-check-hashes-value" }}",
-        "withdrawVerifier":      "{{ index .status.resolvedContractAddresses "zeto-g16-check-inputs-outputs" }}",
-        "batchWithdrawVerifier": "{{ index .status.resolvedContractAddresses "zeto-g16-check-inputs-outputs-batch" }}",
-        "verifier":              "{{ index .status.resolvedContractAddresses "zeto-g16-verifier-anon" }}",
-        "batchVerifier":         "{{ index .status.resolvedContractAddresses "zeto-g16-verifier-anon-batch" }}"
+        "implementation":        '{{`{{ index .status.resolvedContractAddresses "zeto-impl-anon" }}`}}',
```

Comment:
🥸 🥸

Reply:
Yes, the helm templating failed otherwise!